shc/web/widgets.py

```diff
@@ -322,19 +334,28 @@ class ButtonGroup(WebPageItem):
     `Connectable`.
 
     :param label: The label to be shown left of the buttons
-    :param buttons: List of button descriptors
+    :param buttons: List or a List if List of button descriptors
```

Comment:
One afterthought: Can you please add a more detailed description on how to use the parameter, i.e. what effect the single list of button descriptors and the list of lists of button descriptors will have? You can either write it in the parameter description or as a separate paragraph above.